CanvasPlusPlayground/Features/Reminders/ReminderButton.swift

```diff
@@ -0,0 +1,157 @@
+                    .padding([.leading, .trailing], 5)
+                }
+                Section("Custom") {
+                    DatePicker("", selection: $selectedDate, in: Date.now...)
+                    #if os(iOS)
+                        .datePickerStyle(.graphical)
+                    #else
+                        .datePickerStyle(.compact)
+                    #endif
+                }
+            }
+            .navigationTitle("Set a Reminder")
+            #if os(iOS)
+            .navigationBarTitleDisplayMode(.inline)
+            #endif
+            .formStyle(.grouped)
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button("Cancel") {
+                        dismiss()
+                    }
+                }
+                ToolbarItem(placement: .confirmationAction) {
+                    Button("Done") {
+                        scheduleReminder(date: selectedDate)
+                    }
+                }
+            }
+        }
+        .presentationDragIndicator(.visible)
+        .alert(isPresented: $showError) {
+            Alert(title: Text("Error setting reminder"), message: Text(alertMessage))
+        }
+    }
+
+    func scheduleReminder(date: Date) {
+        // Go ahead and strip out the seconds and milliseconds
+        let date = Calendar.current.date(from: Calendar.current.dateComponents([.year, .month, .day, .hour, .minute], from: date)) ?? Date.now
+        Task {
+            do {
+                try await reminderManager.scheduleReminder(for: item, at: date)
+                dismiss()
+            } catch ReminderSchedulingError.invalidDate {
+                alertMessage = "Please select a valid date."
+                showError = true
+            } catch ReminderSchedulingError.notificationsDisbled {
+                alertMessage = "Please enable notifications in your device settings to receive reminders."
+                showError = true
+            }
+        }
+    }
+
+    struct QuickDateButton: View {
+        let imageName: String
+        let imageColor: Color
+        let text: String
+        let date: Date
+        let action: () -> Void
+
+        var body: some View {
+            Button {
+                action()
+            } label: {
+                HStack {
+                    Image(systemName: imageName)
+                        .foregroundColor(imageColor)
+                        .imageScale(.large)
+                        .frame(maxWidth: 40)
+                        .bold()
+                    Text(text)
+                        .bold()
+                    Spacer()
+                    Text(date.dayOfWeekString())
+                        .textCase(.uppercase)
+                        .font(.callout)
+                        .opacity(0.75)
+                }
+
+                .padding([.leading], 5)
+                .padding([.trailing], 10)
+                .padding([.top, .bottom])
```

Comment:
I am unable to click on the empty areas of the button. Add this to fix:
```suggestion
                .padding([.top, .bottom])
                .contentShape(.rect)
```